web/modules/weather_blocks/src/Plugin/Block/Test/EndToEnd/DailyForecast/HourlyData.php.test

```diff
@@ -40,29 +40,90 @@ final class DailyForecastWithHourlyDataTest extends EndToEndBase
     }
 
     /**
-     * Validate that today's hours end at the upcoming 6am.
      * @group e2e
      */
-    public function testTodayHoursEnd6am(): void
+    public function testBeforeMidnightEndsAt6amTomorrow(): void
     {
         $this->onLocationRoute(33.521, -86.812);
 
-        $start = new \DateTimeImmutable();
-        $start = $start->setTimezone(new \DateTimeZone("America/Chicago"));
-
-        $hour = (int) $start->format("H");
-
-        $end = $start->setTime(6, 0, 0, 0);
-        if ($hour >= 6) {
-            $end = $end->modify("+1 day");
-        }
+        // We will look at the weather at 2am point-local time
```

Comment:
Looks to me like 9:15 PM point local time, right? Or am I missing some context where the actual point-local time is something else?

Reply:
Nope, I think that's a copy-pasta mistake.